cli/schedule.go

```diff
@@ -0,0 +1,249 @@
+		},
+	}
+
+	createFlags := []cli.Flag{sid}
+	createFlags = append(createFlags, scheduleSpecFlags...)
+	createFlags = append(createFlags, scheduleStateFlags...)
+	createFlags = append(createFlags, schedulePolicyFlags...)
+	createFlags = append(createFlags, scheduleVisibilityFlags...)
+	createFlags = append(createFlags, removeFlags(flagsForStartWorkflow,
+		FlagCronSchedule, FlagWorkflowIDReusePolicy,
+		FlagMemoKey, FlagMemo, FlagMemoFile,
+		FlagSearchAttributeKey, FlagSearchAttributeValue,
+	)...)
+
+	return []*cli.Command{
+		{
+			Name:        "create",
+			Aliases:     []string{"c"},
+			Usage:       "Create a new schedule",
+			Description: "Takes a schedule specification plus all the same args as starting a workflow",
+			Flags:       createFlags,
+			Action:      CreateSchedule,
+		},
+		{
+			Name:        "update",
+			Aliases:     []string{"up"},
+			Usage:       "Updates a schedule with a new definition (full replacement, not patch)",
+			Description: "Takes a schedule specification plus all the same args as starting a workflow",
+			Flags:       createFlags,
+			Action:      UpdateSchedule,
+		},
+		{
+			Name:  "toggle",
+			Usage: "Pauses or unpauses a schedule",
+			Flags: []cli.Flag{
+				sid,
+				&cli.BoolFlag{
+					Name:    FlagPause,
+					Aliases: FlagPauseAlias,
+					Usage:   "Pauses the schedule",
+				},
+				&cli.BoolFlag{
+					Name:    FlagUnpause,
+					Aliases: FlagUnpauseAlias,
+					Usage:   "Unpauses the schedule",
+				},
+				&cli.StringFlag{
+					Name:    FlagReason,
+					Aliases: FlagReasonAlias,
+					Usage:   "Free-form text to describe reason for pause/unpause",
+					Value:   "(no reason provided)",
+				},
+			},
+			Action: ToggleSchedule,
+		},
+		{
+			Name:  "trigger",
+			Usage: "Triggers an immediate action",
+			Flags: []cli.Flag{
+				sid,
+				overlap,
+			},
+			Action: TriggerSchedule,
+		},
+		{
+			Name:  "backfill",
+			Usage: "Backfills a past time range of actions",
+			Flags: []cli.Flag{
+				sid,
+				overlap,
+				&cli.StringFlag{
+					Name:     FlagStartTime,
+					Usage:    "Backfill start time",
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:     FlagEndTime,
+					Usage:    "Backfill end time",
+					Required: true,
+				},
+			},
+			Action: BackfillSchedule,
+		},
+		{
+			Name:    "describe",
+			Aliases: []string{"d"},
+			Usage:   "Get schedule configuration and current state",
+			Flags: append([]cli.Flag{
+				sid,
+				&cli.BoolFlag{
+					Name:  FlagPrintRaw,
+					Usage: "Print raw data as json (prefer this over -o json for scripting)",
+				},
+			}, flags.FlagsForRendering...),
+			Action: DescribeSchedule,
+		},
+		{
+			Name:    "delete",
+			Aliases: []string{"rm"},
+			Usage:   "Deletes a schedule",
+			Flags: []cli.Flag{
+				sid,
+			},
+			Action: DeleteSchedule,
+		},
+		{
+			Name:    "list",
+			Aliases: []string{"ls"},
```

Comment:
```suggestion
			Aliases: []string{"l"},
```

for consistency with the rest of list operations

Reply:
https://github.com/temporalio/proposals/pull/29/files#diff-996175ce138edb564e46ebdc7849d0e4b6fdbf051dc05246a01ad3c86a7b57aaR22